transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -99,24 +113,78 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
     * @return a Msg object in the desired output format
     */
   def transformMetadata(entityType: String, fileName: String, metadata: Msg): Msg = {
-    // extract info from file name
+    val (entityId, entityVersion) = getEntityIdAndVersion(fileName: String)
```

Comment:
I don't think the type assertion is needed in the argument list
```suggestion
    val (entityId, entityVersion) = getEntityIdAndVersion(fileName)
```

Reply:
Wow 🤔 I really don't know why I did that! Thanks for catching it though!